docs: js counter example does not work

```
git clone https://github.com/tendermint/js-abci.git
cd js-abci/example
npm install
cd ..
node example/counter.js
==>
module.js:550
    throw err;
    ^

Error: Cannot find module 'abci'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/js-abci/example/counter.js:1:86)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
```

node -v v8.14.0
npm -v 6.4.1
js-abci@732db2f5cca02cdccfdb6ab4fd2f6450e6f76923

--- a/docs/app-dev/getting-started.md
+++ b/docs/app-dev/getting-started.md
@@ -256,9 +256,8 @@ You'll also need to fetch the relevant repository, from
 
 ```
 git clone https://github.com/tendermint/js-abci.git
-cd js-abci/example
-npm install
-cd ..
+cd js-abci
+npm install abci
 ```
 
 Kill the previous `counter` and `tendermint` processes. Now run the app:

